junit5/src/main/java/cz/xtf/junit5/listeners/EventsRecorder.java

```diff
@@ -21,7 +26,29 @@
 public class EventsRecorder implements TestExecutionListener {
     @Override
     public void testPlanExecutionFinished(TestPlan testPlan) {
-        final Path eventsLogPath = Paths.get("log", "events");
+        if (OpenShiftConfig.useNamespacePerTestcase()) {
+            return;
+        }
+        recordEvents(Paths.get("log", "events"));
+    }
+
+    @Override
+    public void executionFinished(TestIdentifier testIdentifier, TestExecutionResult testExecutionResult) {
+        if (OpenShiftConfig.useNamespacePerTestcase() && testIdentifier.isTest()
```

Comment:
Is it possible to eliminate `if (OpenShiftConfig.useNamespacePerTestcase` conditions?

E.g. can we record events for each test case even for current state?

Reply:
Can it be set as resolved?

Reply:
Yes, as it was addressed.